services/src/messaging/src/main/java/org/openkilda/messaging/model/FlowDto.java

```diff
@@ -182,6 +186,7 @@ public FlowDto() {
      * @param maxLatency        max latency
      * @param priority          flow priority
      * @param pinned            pinned flag
+     * @param encapsulationType flow encapsulation type
```

Comment:
Should we add it to equals and hashCode? Or to comment above hashcode?

Reply:
Added.